Assets/_BForBoss/_Core/Scripts/Character/PlayerWallRunBehaviour.cs

```diff
@@ -404,9 +369,9 @@ private float GetCameraRoll()
             float targetAngle = 0;
             if (wallDirection != 0)
             {
-                targetAngle = Mathf.Sign(wallDirection) * heading * _maxCameraAngleRoll;
+                targetAngle = Mathf.Sign(wallDirection) * heading * _wallRunData.MaxCameraAngleRoll;
             }
-            return Mathf.LerpAngle(cameraAngle, targetAngle, Mathf.Max(_timeSinceWallAttach, _timeSinceWallDetach) / _cameraRotateDuration);
+            return Mathf.LerpAngle(cameraAngle, targetAngle, Mathf.Max(_timeSinceWallAttach, _timeSinceWallDetach) / _wallRunData.CameraRotateDuration);
```

Comment:
For safety reasons, we should check to see if `_wallRunData.CameraRotateDuration` is 0 first (and if that is the case then divide by 1). 

Alternatively, we can add a min value for `CameraRotateDuration`. Would still prefer if we do the check as well.

Reply:
So default values in struct can only be the built in C# default value which is 0. 

So i'm setting up a Max(cameraiValue, 0.01)